### ISSUE TITLE
Inconsistency with icons

### ISSUE THREAD

**Issue by garuffalo** [strawberrymusicplayer/strawberry#1406]
- [x] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [x] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [x] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
Regardless of whether using a custom icon set, or stock icons in Strawberry, I am seeing odd icons for the "tab close" buttons, and the "Apply; Cancel; OK" buttons in settings. Not a big deal, but it's the little things that drive me crazy.

**To Reproduce**
The problem is in my Strawberry installation on a new, clean install of Linux Mint 21.3. I tried both the PPA and the .deb file version from the Strawberry website. My previous Strawberry install is on an identical computer (Dell e7470 Latitude) -- same specs as the new computer -- same version of Strawberry (PPA) -- same configuration files.

**Expected behavior**
I would expect the new Strawbery install to display the same icons as the identical version on an identical computer, yet the odd icons persist.

**Screenshots:**
![odd_icon_2](https://github.com/strawberrymusicplayer/strawberry/assets/28717740/6c78bea7-ba9b-4900-aaad-e650ea838af1)

![odd_icons_1](https://github.com/strawberrymusicplayer/strawberry/assets/28717740/8666fc13-0e2d-41ad-9ec0-5f37c48e4d02)

![preferred_icons_1](https://github.com/strawberrymusicplayer/strawberry/assets/28717740/ed92e0eb-3e86-40da-bf98-e176505d3dfb)

![preferred_icon_2](https://github.com/strawberrymusicplayer/strawberry/assets/28717740/dd83349b-62a9-43d0-a80c-82f7aafac9b6)

**System Information:**
 - Operating system: Linux Mint 21.3 Cinnamon
 - Strawberry Version: 1.0.23

**Additional context**
I really don't like the old fashioned stock icons in Strawberry, so I have assembled a nice alternate custom set, per instructions on the wiki. It worked perfect on my previous computer.

Interestingly, none of these problem icons seem to be a part of the icons listed on the Github development site. Where is Strawberry pulling them from?

Side note: I previously had an issue (#1178) with using custom icons for the gold and gray "favorite" stars, but that was fixed nicely.

**Comment by jonaski** [comment]
These are standardized buttons in Qt dialogs, strawberry does not specifically set this, they follow the current theme on the system. You need to set the icon theme in KDE Plasma 6 or in qt6ct if not using Plasma 6.

**Comment by jonaski** [comment]
Actually you might be missing a gtk platfiorm theme plugin if you are in using a non Qt desktop.
I suggest to compare the packages on both installations, check for breeze and anything with theme related to Qt or GTK.
Check in Help / About Qt if strawberry is built with Qt 5 or 6.

**Comment by jonaski** [comment]
If you are using Qt 5 check for qt5-style-plugins https://community.linuxmint.com/software/view/qt5-style-plugins, or the equivalent for Qt 6, I assume Linux Mint havent switched to Qt 6 yet?

**Comment by garuffalo** [comment]
I took your advice and compared Qt 5 and Qt 6 packages between installations. My Strawberry is built with Qt 6. Mint has a bunch of Qt 6 related packages, however. I kept installing one after the other, and **libqt6svg6** did the trick. That was installed on the previous computer, but not on the current one.

Makes sense, I guess -- probably related to svg icons? Anyway, now I'm happy. I guess it wasn't a bug -- more of just a behavior. But I really appreciate your help! Thanks!
